### PR TITLE
patches library incorrect size after some actions fix.

### PR DESCRIPTION
**Pull request by imgrby**

### CODE REVIEW

File: browser/scripts/ui.js [hunk 1]
```diff
@@ -71,19 +71,22 @@ VizorUI.prototype.openLoginModal = function() {
 
 
 VizorUI.prototype.onSearchResultsChange = function() {
-	var resultsCount = $('.result.table tbody').children().length;
+	var presetsLib = E2.dom.presetsLib;
+  var resultsCount = $('.result.table tbody').children().length;
```
Comment (kschzt) [inline browser/scripts/ui.js]: indentation

File: browser/scripts/ui.js [hunk 2]
```diff
@@ -71,19 +71,22 @@ VizorUI.prototype.openLoginModal = function() {
 
 
 VizorUI.prototype.onSearchResultsChange = function() {
-	var resultsCount = $('.result.table tbody').children().length;
+	var presetsLib = E2.dom.presetsLib;
+  var resultsCount = $('.result.table tbody').children().length;  
+	var presetsList = presetsLib.find('.preset-list-container');
+	var maxHeight = presetsList.css('maxHeight');
 	if (resultsCount>0) {
 		E2.dom.presetsLib.removeClass('collapsed');
```
Comment (kschzt) [inline browser/scripts/ui.js]: use the presetsLib var

File: browser/scripts/ui.js [hunk 3]
```diff
@@ -71,19 +71,22 @@ VizorUI.prototype.openLoginModal = function() {
...
-		E2.dom.presetsLib.find('.preset-list-container').show();
+		presetsList.show();
 		var resultsHeight = $('.result.table').outerHeight(true);
-		var maxHeight = E2.dom.presetsLib.find('.preset-list-container').css('maxHeight')
 		var newHeight = resultsHeight;
 		newHeight = ( newHeight >= maxHeight ) ? (maxHeight) : (newHeight);
-		E2.dom.presetsLib.height('auto')
-						 .find('.preset-list-container').height(newHeight);
+		E2.dom.presetsLib.height('auto');
```
Comment (kschzt) [inline browser/scripts/ui.js]: use the var :)

**Comment by kschzt** [comment]
otherwise lgtm

File: browser/scripts/ui.js (hunk 1 above)
Comment (imgrby) [inline browser/scripts/ui.js]: fixed

File: browser/scripts/ui.js (hunk 2 above)
Comment (imgrby) [inline browser/scripts/ui.js]: fixed

File: browser/scripts/ui.js (hunk 3 above)
Comment (imgrby) [inline browser/scripts/ui.js]: fixed

**Comment by kschzt** [comment]
go ahead and land this

**Pull request merged**